modules/configuration/php/configuration.class.inc

```diff
@@ -67,7 +67,7 @@ class Configuration extends \NDB_Form
             'Ym'  => 'Ym (EX: 2015-12)'
         ];
 
-        $instruments     = \Utility::getAllInstruments();
+        $instruments     = \Utility::getInstrumentNamesList();
```

Comment:
what's this change necessary for? @CamilleBeau

Reply:
https://github.com/aces/Loris/blob/c70e61511182c7ad8d6ff8d8c92949d03cb2a540/php/libraries/Utility.class.inc#L523

This function is being deprecated & replaced by getInstrumentNamesList